question-servers/freeform.js

```diff
@@ -78,16 +81,6 @@ module.exports = {
                     return callback(null, files);
                 });
             },
-            (files, callback) => {
-                // Filter out any non-directories
-                async.filter(files, (file, callback) => fs.lstat(path.join(sourceDir, file), (err, stats) => {
-                    if (ERR(err, callback)) return;
-                    callback(null, stats.isDirectory());
-                }), (err, results) => {
-                    if (ERR(err, callback)) return;
-                    return callback(null, results);
-                });
-            },
```

Comment:
Why don't we need this anymore? I worry that a `.DS_Store` file or similar will confuse things?

Reply:
Good point, I'm putting the checks back in.